packages/aft/test/version_bump/version_bump_test.dart

```diff
@@ -0,0 +1,228 @@
+      title: 'chore: test',
+      packages: ['amplify_auth_cognito_dart'],
+    ),
+  ],
+  'single_dart_package_fix': [
+    Change(
+      title: 'fix: test',
+      packages: ['amplify_auth_cognito_dart'],
+    ),
+  ],
+  'single_dart_package_feat': [
+    Change(
+      title: 'feat: test',
+      packages: ['amplify_auth_cognito_dart'],
+    ),
+  ],
+  'single_dart_package_breaking': [
+    Change(
+      title: 'feat!: test',
+      packages: ['amplify_auth_cognito_dart'],
+    ),
+  ],
+  'aws_common_fix': [
+    Change(
+      title: 'fix: test',
+      packages: ['aws_common'],
+    ),
+  ],
+  'multi_package_update': [
+    Change(
+      title: 'chore: test secure storage / auth chore',
+      packages: ['amplify_secure_storage', 'amplify_auth_cognito_dart'],
+    ),
+    Change(
+      title: 'fix: test auth fix',
+      packages: ['amplify_auth_cognito', 'amplify_auth_cognito_dart'],
+    ),
+    Change(
+      title: 'feat: test common feat',
+      packages: ['aws_common'],
+    ),
+    Change(
+      title: 'fix: test db common fix',
+      packages: ['amplify_db_common', 'amplify_db_common_dart'],
+    ),
+    Change(
+      title: 'feat: test core/auth feat',
+      packages: ['amplify_core', 'amplify_auth_cognito'],
+    ),
+  ],
+};
+
+// run `dart --define=generate-snapshots=true test ./test/version_bump/version_bump_test.dart --use-data-isolate-strategy`
+// to regenerate snapshots
+const generateSnapshots = bool.fromEnvironment('generate-snapshots');
+
+void main() {
+  group('Version Bump', () {
+    final tempDirectoryPath = '/tmp/${uuid()}';
+    final sourceDirectory = Directory('test/version_bump/data/repo_snapshot');
+
+    for (final t in tests.entries) {
+      final testName = t.key;
+      final changes = t.value;
+      final tempDirectory = Directory('$tempDirectoryPath/$testName');
+      final diffFile = File('test/version_bump/data/snapshots/$testName.diff');
+      group(testName, () {
+        late Repo repo;
+        late String baseRef;
+
+        setUp(() async {
+          await copyDirectory(sourceDirectory, tempDirectory);
+          await GitDir.init(
+            tempDirectory.path,
+            allowContent: true,
+            initialBranch: 'main',
+          );
+          repo = await Repo.open(
+            AftConfigLoader(workingDirectory: tempDirectory).load(),
+          );
+          await runGit(repo, ['add', '.']);
+          await runGit(repo, ['commit', '-m', 'Initial commit\n']);
+          baseRef = await runGit(repo, ['rev-parse', 'HEAD']);
+        });
+
+        tearDown(() async {
+          if (generateSnapshots) {
+            final process = await repo.git.runCommand(['diff', '--unified=0']);
+            final diff = process.stdout as String;
+            await diffFile.writeAsString(processDiff(diff));
+          }
+          await tempDirectory.delete(recursive: true);
+        });
+
+        group('chore', () {
+          test('single amplify package', () async {
```

Comment:
Q: Is `single amplify package` an accurate description for all the tests?

Reply:
No, I should have updated that